setup.cfg

```diff
@@ -0,0 +1,62 @@
+[metadata]
+name = powerfulseal
+version = 2.7.0
+author = Mikolaj Pawlikowski
+author_email = mikolaj@pawlikowski.pl
+description = PowerfulSeal - a powerful testing tool for Kubernetes clusters
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = Apache 2.0
+license_file = LICENSE
```

Comment:
no need to read the licence if you use this format, also specifies the licence type for pypi metadata above